src/dbnode/persist/fs/retriever.go

```diff
@@ -267,7 +267,9 @@ func (r *blockRetriever) fetchBatch(
 	seekerResources ReusableSeekerResources,
 ) {
 	// Resolve the seeker from the seeker mgr
-	seeker, err := seekerMgr.Borrow(shard, blockStart)
+	// TODO(juchan): actually get the correct volume.
+	vol := 0
+	seeker, err := seekerMgr.Borrow(shard, blockStart, vol)
```

Comment:
I don't think this interface should expose the volume. The Borrow() API should just return a seeker for whatever volume the SeekerManager is currently holding a reference to.